src/components/firstProfile/interests/Interests.tsx

```diff
@@ -0,0 +1,474 @@
+import { useEffect, useState } from 'react'
+import {
+  Box,
+  Typography,
+  IconButton,
+  Chip,
+  Dialog,
+  DialogTitle,
+  DialogContent,
+  DialogActions,
+  TextField,
+  Button,
+  Link as MuiLink,
+} from '@mui/material'
+import CloseIcon from '@mui/icons-material/Close'
+import { makeStyles } from 'tss-react/mui'
+import { interestsData as dataInterests } from './interestsData'
+import theme from 'styles/createTheme'
+import LanguageSelector from './languageSelector'
+import InterestsItem from './InterestsItem'
+import { ArrowRightBtn } from './ArrowRightBtn'
+
+type ChipContainerProps = {
+  data: { title: string; item: string[] }
+  multiple?: boolean | undefined
+  onSelectedItems: (selectedItems: string[]) => void
+  selectedItems: string[] | undefined
+}
+
+const Interests = () => {
+  const { classes } = useStyles()
+  const [aboutMe, setAboutMe] = useState('')
```

Comment:
It would be better to use `  const [aboutMe, setAboutMe] = useState(localStorage.getItem('aboutMe') || '')` instead of useEffect ()

The same for selectedLanguages and interestData

Reply:
+done